release/bin/build-release

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env php
+<?php
+
+declare(strict_types=1);
+
+use FINDOLOGIC\FinSearch\Release\ShopwarePluginRelease;
+
+require_once __DIR__ . '/../../vendor/autoload.php';
+
+require_once __DIR__ . '/../src/ReleaseBundler.php';
+require_once __DIR__ . '/../src/ReleaseTagger.php';
+require_once __DIR__ . '/../src/ShopwarePluginRelease.php';
+require_once __DIR__ . '/../src/Utils.php';
```

Comment:
ReleaseBundler, ReleaseTagger, Utils seems unused.

Reply:
No it is not unused, it is used inside `ReleaseTagger`and `ReleaseBundler`, but those should not care about autoloading these files.